definitions/contested/json/AuthorisationCaseEvent/AuthorisationCaseEvent-respSol-nonprod.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "LiveFrom": "01/09/2020",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseEventID": "FR_updateRespondentContactDetails",
+    "UserRole": "[RESPSOLICITOR]",
+    "CRUD": "CRU"
+  },
+  {
+  "LiveFrom": "01/09/2020",
+  "CaseTypeID": "FinancialRemedyContested",
+  "CaseEventID": "FR_updateRespondentContactDetails",
+  "UserRole": "caseworker-divorce-financialremedy-judiciary",
+  "CRUD": "CRU"
```

Comment:
These should be 
"CRUD": "R"
for all except [RESPSOLICITOR] who is CRU. As only RESPSOLICITOR should be able to trigger this event, the others only need R for seeing it in history